.azure/gpu-tests-pytorch.yml

```diff
@@ -155,6 +163,7 @@ jobs:
         PL_RUN_CUDA_TESTS: "1"
         PL_STANDALONE_TESTS_SOURCE: "pytorch_lightning"
       displayName: 'Testing: PyTorch standalone tests'
+      condition: eq(variables['PACKAGE_NAME'], 'pytorch')
```

Comment:
Why add this? Standalone should still work

Reply:
was not sure is we wanted to run it, so just left it for the stand-alone package...
feel free to send update :chipmunk: